docs/source/author/topics.rst

```diff
@@ -0,0 +1,685 @@
+Topics application
+##############################################################################
+
+The topics application is the main focus of the CS Unplugged website, as it
+contains the majority of educational material for the project.
+
+.. contents:: Contents
+  :local:
+
+Topics overview
+==============================================================================
+
+A general overview of the topics application can be described in the following
```

Comment:
Done (but changes don't make this line out of date). I added link after the first usage of application (several lines above the occurrence linked).